e2e/helpers/verification.ts

```diff
@@ -0,0 +1,74 @@
+import { Page, expect } from '@playwright/test'
+import { navigateToHomepageIfPageHasNotBeenLoaded } from './localStorage'
+
+export async function waitForSrcEndpointConfig(page: Page) {
+  // window only available after page has initially loaded
+  await navigateToHomepageIfPageHasNotBeenLoaded(page)
+  // ensure that endpoint config is set,
+  // ...so API calls point to the correct stack
+  await expect(async () => {
+    const response = await page.evaluate('window.SRC.OVERRIDE_ENDPOINT_CONFIG')
+    expect(response).not.toBeUndefined()
+  }).toPass()
+}
```

Comment:
A page can be ready for PW interaction before the SRC endpoint config override is set, so await the expected override so API calls point to the expected backend stack